app/controllers/application_controller.rb

```diff
@@ -111,4 +112,13 @@ def staff?
   def true_user
     @true_user ||= User.find_by(id: session[:user_id])
   end
+
+  def display_error_page(exception)
```

Comment:
Can we make this method name more descriptive? The ApplicationController can throw a few different errors, this method is specific to 404 errors.